### PR TITLE
[SN-107]Added support for annotation import with img bytes

### PR DESCRIPTION
**Pull request by ramy1951**
Updated notebooks to include support for bytes array for annotation import on masks.

### CODE REVIEW

File: examples/annotation_import/image.ipynb [hunk 1]
```diff
@@ -1,18 +1,16 @@
 {
```
[the remark this answers is not in the document]
Comment (ramy1951) [reply examples/annotation_import/image.ipynb:1]: <p>I fixed it, thanks</p>

**Comment by ramy1951** [comment]
Screenshot 2024-01-26 at 10 54 11 AM
Here is the project I created using the notebook

File: examples/annotation_import/image.ipynb (hunk 1 above)
Comment (ovalle15) [inline examples/annotation_import/image.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #31.          "name": " ",
</pre><p>This should be "mask" not "&lt;insert segementation mask name as seen in ontology&gt; ",</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (ovalle15) [inline examples/annotation_import/image.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #33.          "dataRow": {
</pre><p>This shouldn't be place here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (ovalle15) [inline examples/annotation_import/image.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #17.          "dataRow": {
</pre><p>Also no need to include this here </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

**Comment by ovalle15** [comment]
@ramy1951 thank you for taking a look, can you please use the format for all the other annotations and always run the notebooks with both python annotations and ndjson to verify your work. Thank you !

**Comment by ovalle15** [comment]
@ramy1951 any chance we can get this merge today ?

**Comment by ramy1951** [comment]
Addressed reviews, I tried the notebook after updates with all annotations. It works just fine

File: examples/annotation_import/image.ipynb (hunk 1 above)
Comment (ramy1951) [reply examples/annotation_import/image.ipynb:1]: Done, I was trying to add instructions for users instead of making it demo-able

Comment (ramy1951) [reply examples/annotation_import/image.ipynb:1]: I did not realize that they are overridden later.

**Review by ovalle15 — APPROVED** [review]

**Pull request merged**